rootfs/etc/cron.daily/dietpi

```diff
@@ -45,6 +45,9 @@
 
 	fi
 	#----------------------------------------------------------------
+	#Clear DietPi motd
+	[[ -f /tmp/.dietpi_motd ]] && rm /tmp/.dietpi_motd &
+	#----------------------------------------------------------------
```

Comment:
@Fourdee 
Perhaps download the new MOTD here as well? So the banner can show (execute) faster on first prompt after daily cron.

Reply:
Implemented:
- https://github.com/MichaIng/DietPi/commit/26cffb70639309df6c9681ff10cf16c4c2fbb2dd
And MOTD is now a source-able script, so we can print targeted messages based on system/setup/software:
- https://github.com/MichaIng/DietPi/commit/5245a6c39d5b001ac9f70cc172a26db33a04a32f
- [x] Updated `motd` file on server